paper.tex

```diff
@@ -257,6 +257,32 @@ \subsection*{Population genetics model}
 over that interval, which can be a constant value or function defined by start
 and end sizes that must remain positive.
 
+\krtcomment{The paragraph(s) below subsume(s)/replace(s) some of what appears in later pargraphs.
+	I'm not making an attempt yet to revise that later content--I think other authors should
+	weigh in.}
+
+\krtcomment{Aaron -- I have perhaps over-simplified the description of marginal allele frequency dynamics?}
+
+Within a population, we implicitly assume that allele frequency dynamics can
+be described by the "Wright-Fisher" model.  Briefly, generations are non-overlapping (all parents
+reproduce and die simultaneously) and the expected frequency (at birth) of an allele currently at frequency $p_i$
+one generation in next generation is $p_iw_i/\bar{w}$, where $w_i$ and $\bar{w}$ are the marginal and
+mean fitnesses, respectively, properly weighted according to ancestry proportions. In this framework,
+a forward-time simulation of finite populations is equivalent to multinomial sampling of allele frequencies each
+generation (\citet[][pp 29-31]{burger2000-ul}, \citet[][pp 179-181]{crowkimura1970})
```

Comment:
I'm confused. How are the fitness effects taken into account if one just does multinomial sampling of allele frequencies?

Reply:
Using E[pi] as defined as the coefficients. The slow way to do a sim is to calculate marginal allele fitnesses and then call rmultinom(ps, N).

Reply:
At birth, f(AA) i p^2.  After selection, it is f(AA)' = p^2wAA/wbar.  Likewise f(Aa)' = 2pawAa/wbar.  E[p] at time t + 1 is f(AA)' + f(Aa)'/2, etc., and it has a binomial number of offspring.We don't want to write all this as it is in the citations.